### PR TITLE
refactor(favorite): MP-363 Add favorite card item to library

### PR DESCRIPTION
**Pull request by args06**
## Update Items
- [x] [[MP-363](https://args06.atlassian.net/browse/MP-363)] Finalize Library UI Layout

## Summary Description
- Add Favorite Card Item
- Change library card type programmatically for favorite purpose

### After
- 
image

## *Type of Change
- [ ] docs: Documentation only changes, README.md, Wiki
- [ ] feat: A new feature
- [ ] refactor: A code change that neither fixes a bug nor adds a feature
- [ ] test: Anything related to unit/integration testing
- [ ] revert: Revert a commit

## Type of Scope
- [ ] album: Changes that affect user packages
- [ ] music: Changes that affect music packages
- [ ] account: Changes that affect account packages
- [ ] favorite: Changes that affect favorite packages
- [ ] playlist: Changes that affect playlist packages
- [ ] playlist-detail: Changes that affect playlist-detail packages
- [ ] music-artist: Changes that affect music-artist packages
- [ ] general: all stuff except above

### CODE REVIEW

**Comment by RTAgung** [comment]
Give the screeshot please

File: app/src/main/java/com/example/nuberjam/ui/main/library/LibraryFragment.kt [hunk 1]
```diff
@@ -51,4 +53,14 @@ class LibraryFragment : Fragment() {
             }
         }
     }
+
+    private fun setFavoriteItem() {
+        binding.apply {
+            favoriteItem.tvPlaylistType.text = getString(R.string.favorite)
```
Comment (args06) [inline app/src/main/java/com/example/nuberjam/ui/main/library/LibraryFragment.kt:61]: Here i change the card type to "Favorite"

**Comment by args06** [comment]
> Give the screeshot please

For the screenshot is still the same, because i changed the text programmatically

File: app/src/main/java/com/example/nuberjam/ui/main/library/LibraryFragment.kt (hunk 1 above)
Comment (RTAgung) [reply app/src/main/java/com/example/nuberjam/ui/main/library/LibraryFragment.kt:61]: just for tvType? how about tvName?

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -43,6 +43,7 @@
     <string name="now_playing_title">Now Playing</string>
     <string name="classic_song">Classic Song</string>
     <string name="playlist">Playlist</string>
+    <string name="favorite">Favorite</string>
```
Comment (RTAgung) [inline app/src/main/res/values/strings.xml:46]: The text is "Favorite" or "Liked Song"?

Comment (args06) [reply app/src/main/res/values/strings.xml:46]: done

Comment (args06) [reply app/src/main/res/values/strings.xml:46]: that is favorite, and for like song i already add it on my new commit

**Review by RTAgung — APPROVED** [review]

**Pull request merged**